VisualInstructionComponentTests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+import MapboxDirections
+
+class VisualInstructionComponentTests: XCTestCase {
+    func testJSONInitialization() {
+        let component = VisualInstructionComponent(json: [
+            "text": "Take a hike",
+            "imageURL": "",
+        ])
+        XCTAssertEqual(component.text, "Take a hike")
+        XCTAssertNil(component.imageURL)
```

Comment:
I'm a little confused by this. Your test passes even with the code change reverted.

Reply:
The test was wrong. Fixed in f4a6425c8abe7357eaf89692c32cbba78ee9b112.